municipal_finance/models.py

```diff
@@ -383,12 +402,31 @@ class UifwexpFacts(models.Model):
 
     class Meta:
         db_table = 'uifwexp_facts'
-        unique_together = (('demarcation_code', 'financial_year', 'item_code'),)
+        unique_together = (
+            ('demarcation_code', 'financial_year', 'item_code'),)
 
 
 class DemarcationChanges(models.Model):
     date = models.DateField(blank=False, null=False)
     old_code = models.TextField(blank=False, null=False, db_index=True)
     new_code = models.TextField(blank=False, null=False, db_index=True)
-    old_code_transition = models.TextField(blank=False, null=False, db_index=True)
-    new_code_transition = models.TextField(blank=False, null=False, db_index=True)
+    old_code_transition = models.TextField(
+        blank=False, null=False, db_index=True)
+    new_code_transition = models.TextField(
+        blank=False, null=False, db_index=True)
+
+
+class MunicipalityProfilesRebuild(models.Model):
+    user = models.ForeignKey(User)
+    datetime = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'municipality_profiles_rebuild'
+
+
+class MunicipalityProfile(models.Model):
+    demarcation_code = models.CharField(max_length=10, primary_key=True)
```

Comment:
could also do a foreign key on geography but not a big deal

Reply:
I like that idea. Should we perhaps give this model an `id` for the primary key and change to a foreign key, or use a `OneToOneField`?